lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -343,13 +343,13 @@ def call(params)
           results.compact!
           counts = results.group_by { |r| r[:type] }.map { |g, r| [g, r.count] }
           counts.each do |category, count|
-            puts "There were #{count} events of type #{category}"
+            GoodData.logger.info("There were #{count} events of type #{category}")
           end
           errors = results.select { |r| r[:type] == :error || r[:type] == :failed }
           return if errors.empty?
 
-          puts 'Printing 10 first errors'
-          puts '========================'
+          GoodData.logger.info('Printing 10 first errors')
+          GoodData.logger.info('========================')
           pp errors.take(10)
```

Comment:
shouldn't this be redirected to the logger too?

Reply:
Fixed.